media/css/m24/components/footer-refresh.scss

```diff
@@ -40,7 +40,7 @@ $max-footer-content-width: $content-max;
     flex-direction: column;
     gap: $grid-gutter;
 
-    @media(min-width: $screen-lg) {
+    @media #{$mq-lg} {
```

Comment:
Thanks for these ❤️  — The same is in footer-newsletter if you fancy taking a look too.